src/models/GitConfig.js

```diff
@@ -43,13 +43,15 @@ const schema = {
 
 const isSection = line => line.trim().startsWith('[')
 
-const extractSection = line =>
-  line
+const extractSection = line => {
+  const indices = [line.indexOf(']'), line.indexOf(' ')].filter(i => ~i)
```

Comment:
Any particular reason you went with `(i => ~i)` rather than `(i => i > -1)`? It took me a while to understand what that was doing (yay 2's-complement arithmetic).

Reply:
No, I like it because it is cleverer than me, but sometimes fail to keep it out of non-personal projects. :-) Fix incoming.

Reply:
Haha, whew! I was secretly worried that this was some common knowledge trick that everybody knew but me.